src/react/views/websocket/components/Server/index.tsx

```diff
@@ -8,26 +8,27 @@ import WebSocketContext, { IWebSocketContext } from '../../context/websocket';
 const INITIAL_VALUE = 'ws://127.0.0.1:5200';
 
 function Server(): JSX.Element {
-  const { connect } = React.useContext<IWebSocketContext>(WebSocketContext);
-  const [value, setValue] = React.useState<string>(INITIAL_VALUE);
+  const { connect, isConnected } = React.useContext<IWebSocketContext>(WebSocketContext);
+  const [value, setValue] = useState<string>(INITIAL_VALUE);
 
-  const isConnectDisabled = React.useMemo(() => {
+  const isConnectDisabled = useMemo(() => {
```

Comment:
This validation is temporal. A hardcoded URL is being used in order to test the application with websockets, will be removed when development process of websocket view is done. @Sansossio